Cargo.toml

```diff
@@ -21,7 +21,8 @@ pending_review_screen = ["ledger_device_sdk/pending_review_screen"]
 [package.metadata.ledger]
 curve = ["secp256k1"]
 flags = "0"
-path = [""]
+path = ["44'/1'"]
+name = "Rust Boilerplate"
```

Comment:
Do we want to align the repo name with the appname ?

Reply:
Not really. `"Rust Boilerplate"` is the app name as displayed on the device dashboard, the repo name is not very suitable for that with the dashes... Not very good looking.

Reply:
I mean rename the repo to `app-rust-boilerplate`

Reply:
Don't think we need to do that :man_shrugging: